src/pages/AddLiquidityV2/index.tsx

```diff
@@ -407,91 +472,119 @@ export default function AddLiquidity() {
     setTxHash('')
   }, [navigate, networkInfo.route, onFieldAInput, txHash])
 
-  const addIsUnsupported = false
-
-  // const clearAll = useCallback(() => {
-  //   onFieldAInput('')
-  //   onFieldBInput('')
-  //   onLeftRangeInput('')
-  //   onRightRangeInput('')
-  //   history.push(`/add`)
-  // }, [history, onFieldAInput, onFieldBInput, onLeftRangeInput, onRightRangeInput])
-
-  // get value and prices at ticks
-  const { [Bound.LOWER]: tickLower, [Bound.UPPER]: tickUpper } = ticks
-  const { [Bound.LOWER]: priceLower, [Bound.UPPER]: priceUpper } = pricesAtTicks
+  const mountRef = useRef(false)
+  const handleDismissConfirmationRef = useRef(handleDismissConfirmation)
+  useEffect(() => {
+    if (ENV_LEVEL > ENV_TYPE.LOCAL && !mountRef.current) {
```

Comment:
why do we need this if?

Reply:
It helps me when dev. It prevent clear redux state when reload/hot-reload the page. But for prod, it should clear the state when leave and reload the page.